src/lib/input/README.md

```diff
@@ -78,7 +78,25 @@ The divider (line under the `<md-input>` content) color can be changed by using
 
 You can label the `<md-input>` as you would a regular `<input>`.
 
+## Focus
 
+You can put the focus on an input component using the `focus()` method.
+
+### Example
+
+```html
+<md-input #nameInput placeholder="name"></md-input>
```

Comment:
You're right, I used the ViewChild incorrectly. What I wanted to write was instead:
@ViewChild('nameInput') nameInput: MdInput;
to that this sample still works when multiple md-input are present in the template.
Do you agree?